src/api/checkout.js

```diff
@@ -24,4 +24,18 @@ app.post('/', (req, res) => {
     });
 });
 
+app.post('/longterm', (req, res) => {
+    addAction('NEW_LONGTERM_CHECKOUT', {
```

Comment:
Shouldn't there be a test that covers this? Code coverage says that the other api method is tested, so shouldn't this one be?
https://codecov.io/gh/TheFourFifths/consus/src/new-checkout-tff153/src/api/checkout.js